### ISSUE TITLE
pytest failed when build mmocr with "full script",  installed mmcv==1.3.0, but all tests failed.

### ISSUE THREAD

**Issue by Bovey0809** [open-mmlab/mmocr#76]
pytest  showed the fiailed "mmcv should be >=1.2.4 <=1.3.0", but the full script will install mmcv-full with version 1.3.0, 
fixed it by adding ```pip install mmcv==1.2.6``` in the 'full script'
```bash
conda create -n open-mmlab python=3.7 -y
conda activate open-mmlab

# install latest pytorch prebuilt with the default prebuilt CUDA version (usually the latest)
conda install pytorch==1.5.0 torchvision==0.6.0 cudatoolkit=10.1 -c pytorch

# install the latest mmcv-full
pip install mmcv-full==1.2.6

# install mmdetection
pip install mmdet==2.9.0

# install mmocr
git clone https://github.com/open-mmlab/mmocr.git
cd mmocr

pip install -r requirements.txt
pip install -v -e .  # or "python setup.py build_ext --inplace"
export PYTHONPATH=$(pwd):$PYTHONPATH
```

**Comment by jeffreykuang** [comment]
you can create a pr to fix it. Thanks

**Comment by innerlee** [comment]
> but all tests failed

Hi, this is abnormal. Currently there is a COCO dependency issue, but that affects only one test, See the recent CI log here https://github.com/open-mmlab/mmocr/pull/74/checks?check_run_id=2351789972

![image](https://user-images.githubusercontent.com/9464825/114959974-b2634580-9e98-11eb-8ad8-4356b5de974e.png)

Please post the full error message so that we can debug :)

**Comment by cuhk-hbsun** [comment]
> pytest showed the fiailed "mmcv should be >=1.2.4 <=1.3.0", but the full script will install mmcv-full with version 1.3.0,
> fixed it by adding `pip install mmcv==1.2.6` in the 'full script'
> 
> ```shell
> conda create -n open-mmlab python=3.7 -y
> conda activate open-mmlab
> 
> # install latest pytorch prebuilt with the default prebuilt CUDA version (usually the latest)
> conda install pytorch==1.5.0 torchvision==0.6.0 cudatoolkit=10.1 -c pytorch
> 
> # install the latest mmcv-full
> pip install mmcv-full==1.2.6
> 
> # install mmdetection
> pip install mmdet==2.9.0
> 
> # install mmocr
> git clone https://github.com/open-mmlab/mmocr.git
> cd mmocr
> 
> pip install -r requirements.txt
> pip install -v -e .  # or "python setup.py build_ext --inplace"
> export PYTHONPATH=$(pwd):$PYTHONPATH
> ```

The install.md has beed updated. Please follow `install.md` to install the mmdet and mmcv with latest version.
```bash
pip install mmcv-full==1.3.1
pip install mmdet==2.11.0
```
